pkg/service/service.go

```diff
@@ -977,6 +977,20 @@ func (s *Service) GetServiceNameByAddr(addr lb.L3n4Addr) (string, string, bool)
 	return svc.svcNamespace, svc.svcName, true
 }
 
+// GetServiceIDByAddr returns the LB ID of the service. The second return value is set to true
+// if and only if the service is found in the map.
+func (s *Service) GetServiceIDByAddr(addr lb.L3n4Addr) (lb.ID, bool) {
```

Comment:
Where is this called?

Reply:
Ah good catch, this is actually left over from my previous implementation, removed. (Unless we need to preserve the original service map entry id for some reason?)

Reply:
I don't see a reason to preserve the id, but I refer this to @aanm.